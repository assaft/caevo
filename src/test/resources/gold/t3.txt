An earthquake struck Los Angeles three years ago yesterday.