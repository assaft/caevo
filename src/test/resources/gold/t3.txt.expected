ei1,struck
t1,1966-07-20
l1,ei1,t1,IS_INCLUDED